pkg/apk/apkindex.go

```diff
@@ -127,9 +136,9 @@ func ParsePackageIndex(apkIndexUnpacked io.Reader) ([]*Package, error) {
 		case "U":
 			pkg.URL = val
 		case "D":
-			pkg.Dependencies = strings.Split(val, " ")
+			pkg.Dependencies = splitRepeatedField(val)
 		case "p":
-			pkg.Provides = strings.Split(val, " ")
+			pkg.Provides = splitRepeatedField(val)
```

Comment:
Thanks, can you do this for `InstallIf` as well?

Reply:
Done, thanks.